---
title: "About Me"
author: "Michael Franc Antolin"
output:
  html_document:
    toc: yes
  word_document:
    toc: yes
  pdf_document:
    toc: yes
bibliography: references.bib
editor_options: 
  markdown: 
    wrap: 72
---



# Who I Am and Where I Came From (like from PA and Canada or something)

A CHANGED VERSION JUST TO CHECK RSTUDIO TO GITHUB CONNECTIONS 


I arrived at Colorado State University in 1992 (and am originally from rural PA near Gettysburg), after studying biology at the [University of Pennsylvania, BA 1981](https://www.bio.upenn.edu/), The [University of Alberta, Canada, M.Sc. 1985](https://www.ualberta.ca/biological-sciences/index.html), and [Florida State University, PhD 1990](https://www.bio.fsu.edu/)).  I am currently Professor and former department Chair of the [Biology Department](https://www.biology.colostate.edu/).

I consider myself an evolutionary and population biologist, and have taught evolution and population genetics at both undergraduate and graduate levels.  Currently I'm greatly enjoying teaching LIFE 102 (Attributes of Living Systems) to Honors students.   

This life was a natural for me. We moved to rural PA from Canada when I was three years old, to a former egg farm on the Mason Dixon Line (the property boundary has one of the marble markers from the original survey in 1763-1767). I was partly feral as a kid, spending much time outdoors in the fields and forests surrounding our house and the nearby Tom's Creek, in view of the Blue Ridge Mountains.  My father was a philosophy professor who planted the seeds of skepticism in my mind, along with a love for Nature and the trees we transplanted to the property.  All are more-or-less still in place.


As part of my public outreach to science education, I was a charter member of the “Colorado Citizens for Science” advocacy group (now defunct), which supported teachers by tracking curriculum and science teaching standards for CO public schools.  I have written several articles on the “creation-evolution debates” and on science education for the general public.  More recent writings have been on the relationship between evolution and medicine, and on the evolutionary history of the Darwin family of doctors [@antolin2011evolution].    


I was fortunate to be part of design and construction of the new Biology Building, a 150,000 square foot comprehensive education and research facility. The Biology Building gives Biology a distinct home, a destination for hands-on learning in both classrooms and study areas, and state-of-the-art research laboratories.  The [Biology Building](http://www.biology.colostate.edu/new-building/) opened in Fall 2017. 


My top-four favorite leisure activities:

1.  Playing hockey
2.  Gardening
3.  Skiing, both downhill at resorts and touring on Cameron Pass
4.  Doom-scrolling the internet, sometimes about railroad and mining history in Colorado 


Here are a few pictures of me, first with my family on the front porch in Toronto in 1961 (I'm on the bottom stair), and me doing field work on Mule deer in Red Mountain Open Space about 10 years ago: 

```{r me_pic, echo=FALSE, out.width="500px"}
knitr::include_graphics(c("images/Antolincki Toronto 1962.jpg","images/Antolin with deer.jpg"), auto_pdf = TRUE)
```


# Research Interests


My research is focused on evolution and genetics of infectious diseases, which also leads to thinking about how evolution may be applied to the medical sciences.

People in my lab have focused on the epidemiology of plague in Black-tailed prairie dogs, summarized in @salkeld2016ecological. This is the same bacterial pathogen responsible for massive human epidemics in medieval Europe and Asia, what’s known as the Black Death in the 1300s.  Native to Asia, plague was accidentally introduced into the western U.S. in the early 1900s and now decimates wild rodent populations throughout the western half of the U.S., along with 10-15 human cases per year. Along with studies of prairie dog population structure [@roach2001genetic], studies of transmission by fleas [@wilder2008oropsylla; @tripp2010flea], we used a SNP-chip to demonstrate genetic identity of plague clones in CO relative to other samples from the western U.S.: [@lowell2015single].   

More recently we also started work on the genetics of chronic wasting disease in mule deer northern Colorado.  This neurodegenerative disease is similar to the “mad cow disease” in Europe, and is caused by a mis-folded infectious protein called a prion.  Again, we use genetic tools to trace transmission pathways (microsatellites and RAD-seq, unpublished).

My research tends to be interdisciplinary and collaborative, and I have the great fortune of having many colleagues here at Colorado State and with other agencies Fort Collins to help us understand epidemiology and evolution of infectious diseases in natural populations.

## Influential Papers


My favorite paper in all of evolutionary biology is @gould1979philosophical "Spandrels of San Marco and the Panglossian Paradigm."  This rhetorical masterpiece examines how to study adaptation, with the idea that many traits in organisms may not be adaptations per se, but may arise as byproducts of  phylogenetic constraints, contingent on the body plans within particular lineages.  This also fits with my view of genetic neutrality, coalescence, etc.


Recently I was a co-organizer an of a study group at the National Institute for Mathematical and Biological Synthesis  [NimBIOS](https://legacy.nimbios.org/) in Knoxville, Tennessee that resulted in two papers on using genomic methods to study adaptation in wild populations  [@hoban2016finding;@lowry2017breaking].  The articles seem to be getting a lot of attention, both positive and negative.  


## The Mathematics Behind My Research

Much of my work is statistical, and of course one of the most commonly used statistics in population genetics are the Fixation Indices, especially $F_{st}$.

Here are two versions of $F_{st}$, one calculated from heterozygosity and one based on the variance in  allele frequencies among populations.

First, based on heterozygosity: 
$$
F_{st} = \frac{H_T - H_S}{H_T}
$$
Where 

|     $H_{T}$ is the total heterozygosity across all populations, and

|     $H_{S}$ is the average heterozygosity within populations.

|
|
Second, from variance in allele frequencies among populations:
$$
F_{st} = \frac{Var_p - \bar{p}(1 - \bar{p})}{Var_p}
$$
Where


|     $p$ is frequencies of an allele in a series of populations,

|     $Var_{p}$ is the variance of allele frequencies across populations, and 
|     $\bar{p}$ is the average allele frequency across all populations.

|
|
A second equation that we use measures the mean kinship $\bar{k}_i$ of each individual $i$ (based on relatedness between individuals calculated from pedigrees) with the $n-1$ other individuals in a captive-breeding population.  


$$
\bar{k}_i = \sum_{j=1,j\neq i}^{n} \frac{2f_{ij}}{n-1}
$$
Where 

|    $f_{ij}$ is the inbreeding coefficient of the offspring of two individuals in a population, 
|    and $2f_{ij}$ is the relatedness between the two individuals, for the $n$ individuals in the pedigree.

|
|
I used ChaptGPT to get some base code for the equations, but modified and upgraded.   


## My Computing Experience

I learned to program in FORTRAN back in 1983.  This was useful for the logical necessities of coding in other programs, but was also usefule for modifying and running command-driven versions of BIOSYS (before GUI's etc.).  I have used R since abut 2004, and in 2005 took a spatial statistics course that used R (S-plus actually, but let's not quibble).  

My snippet of R is below in the "Evaluating some R code section below).  


A snippet of FORTRAN that asks you to input your name and print it out:

1.  program namer
2.  implicit none
3.  character::name
4.  print*, 'What is Your name ? '
5.  read*, name
6.  print*, 'Your Name Is',' ', name
7.  end program namer


## What I hope to get out of this class

Three thing I desire:

-   To re-familarize myself with using programs to carry out population genomic analyses.
-   To update my abilities to manipulate sequence data, including complete genomes and reduced representation methods like RAD-seq.  
-   To apply these to data that we have for Mule deer (Rad-seq) and for the Black-footed ferret (complete genomes).  

# Evaluating Some R Code

This is R-code written to analyze some immunological assays for testing a plague vaccine in Black-footed ferrets.  This was part of Debbie McCuen's MS thesis from 2019.   


```{r, message=FALSE}

# Debbie's PRNT analysis, original data in BFF vaccinia PRNT data updated v7.xlsx

#  clean out your memory before beginning the analysis
rm(list=ls(all=TRUE))

#  The data
vacciniaHI = c(43.48, 78.26, 16.54, 23.19, 23.19, 23.19, 40.58, 84.51, 15)
vacciniaNONHI = c(46.38, 8.7, 2.02, 14.49, 14.49, 5.8, 26.09, 28.17, 0.19)
F1VHI = c(55.77, 41.15, 36.92, 32.5, 29.42, 27.88, 33.08, 21.54, 21.54)
F1VNONHI = c(1.79, 0.58, 6.35, 0.64, 2.47, 4.2, 29.49, 1.79, 3.04)

#Paired t-test for vaccinia group - treatment is HI vs non-HI, hypothesize means are equal
t.test(vacciniaHI, vacciniaNONHI, paired = T)

#Paired t-test for F1V group - treatment is HI vs non-HI, hypothesize means are equal
t.test(F1VHI, F1VNONHI, paired = T)

#  ?? 2 sample t-test between F1VHI and VacciniaHI - hypothesize means are equal
t.test(vacciniaHI, F1VHI)

# 2 sample t-test between F1VNONHI and VacciniaNONHI, confidence interval shows the difference btw the 2, ##hypothesize means are equal
t.test(vacciniaNONHI, F1VNONHI)

# Is distribution normal?
hist(vacciniaHI, xlim=c(0,90), ylim=c(0, 7))
hist(F1VHI, xlim=c(0,90), ylim=c(0,7))
hist(vacciniaNONHI, xlim=c(0,90), ylim=c(0,7))
hist(F1VNONHI, xlim=c(0,90), ylim=c(0,7))

```

# Citations
